modules/geopackage/package.json

```diff
@@ -24,7 +24,7 @@
     "README.md"
   ],
   "dependencies": {
-    "@loaders.gl/core": "3.0.0-alpha.19",
```

Comment:
Seems like you still need the `core` dependency? It imports `parseSync` from `core` to handle WKB parsing.

Reply:
Yes you can add it back for now. 

But loader modules are not supposed to depend on core, they get `parse` injected through the loader context instead.